themes/bootstrap3/templates/Recommend/SideFacets/checkbox-filters.phtml

```diff
@@ -1,9 +1,16 @@
 <?php foreach ($checkboxFilters as $current): ?>
-  <a class="checkbox-filter<?php if(!($results->getResultTotal() > 0 || $current['selected'] || $current['alwaysVisible'])): ?> hidden<?php endif; ?>" href="<?=$current['selected'] ? $results->getUrlQuery()->removeFilter($current['filter']) : $results->getUrlQuery()->addFilter($current['filter']);?>">
-    <?php if ($current['selected']): ?>
-      <span class="sr-only"><?=$this->transEsc('clear_tag_filter') ?></span>
-    <?php endif; ?>
-    <i class="fa fa-<?=$current['selected'] ? 'check-square-o' : 'square-o' ?>"></i>
-    <?=$this->transEsc($current['desc']) ?>
-  </a>
+  <div class="facet" >
+    <a class="checkbox-filter<?php if(!($results->getResultTotal() > 0 || $current['selected'] || $current['alwaysVisible'])): ?> hidden<?php endif; ?>" href="<?=$current['selected'] ? $results->getUrlQuery()->removeFilter($current['filter']) : $results->getUrlQuery()->addFilter($current['filter']);?>" data-facet="<?=$this->escapeHtmlAttr($current['filter'])?>" data-facet="<?=$this->escapeHtmlAttr($current['filter'])?>">
```

Comment:
I just fired this up on my test server. I think the data-facet you're adding to the checkbox filters is cause your JS to confuse checkbox filters for facet-lists. It's working great everywhere else.

![image](https://user-images.githubusercontent.com/451107/47442992-6af51600-d781-11e8-9718-34f147b1be93.png)

Reply:
I couldn't replicate the issue right away, so could you share your facets.ini? I'll make the selectors more specific, but it's hard to verify without being able to reproduce.